README.md

```diff
@@ -252,6 +252,10 @@ mail:
     encryption: tls
     auth_mode: ~
 ```
+### [Running behind a trusted proxy](#run-trusted-proxy)
+
+If you are running open cfp behing a proxy server which adds X-Forwarded-For headers (this could be a cloud based load balancer or a service such as cloudflare) you will need to set the environment variable TRUST_PROXIES to true this will ensure that opencfp trusts the headers set by these proxies for the original IP address and ssl mode. Setting this will trust these headers regardless of where the original request originates, so it's advisable to either lock down your instance so that only the trusted proxy can access it, or modifiy the list of trusted proxies in the index.php file to only include the ip addresses of your proxies.
```

Comment:
The project name is OpenCFP

Reply:
Also: `modifiy` should be `modify`

Reply:
Apologies for chiming it, but I think it also should be `Cloudflare`, see https://www.cloudflare.com.